docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
-| Labels | What you add to each region while labeling a task in Label Studio. |
-| Relation | A defined relationship between two labeled regions. |
-| Result | A label applied to a specific region as stored in an annotation or prediction. See [Label Studio JSON format of annotated tasks](export.html#Label-Studio-JSON-format-of-annotated-tasks). |
-| Predictions | What machine learning models create for an unlabeled dataset. |
-| Pre-annotations | Predicted annotations in Label Studio format, either in a file or from a machine learning backend. See [import pre-annotations](predictions.html).
-| Annotations | The output of a labeling task. Previously called "completions". |
-| Templates | Example labeling configurations that you can use to specify the type of labeling that you're performing with your dataset. See [all available templates](/templates) |
-| Tags | Configuration options to customize the labeling interface. See [more about tags](/tags). |
+When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table defines the terms that appear throughout Label Studio.
+
+**Annotation**
+Annotation is the process of labeling data in various formats such as video, images, or text, so that machines can understand it. The output of a labeling task which was previously called **Completions**. 
+
+**Artificial Intelligence (AI)**
+AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans.  It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions.
+
+**Audio classification**
+Audio classification or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text to speech applications.
+
+**Audio labeling**
+Audio labeling or speech labeling is a standard annotation technique that concerns separating sounds and labeling with specific metadata.
+
+**Audio tagging**
+Audio tagging is a task to predict the tags of audio clips. This includes music tagging, acoustic scene classification, audio event classification, and so on.
+
+**Audio transcription**
+Audio transcription transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.
+
+**Automatic Speech Recognition (ASR)**
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
+A bounding box describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner.
```

Comment:
A [bounding box](/image_bbox.html) describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner. For more information, see the [labeling configuration for object detection with bounding boxes](/image_bbox.html#Labeling-Configuration).